src/app/login-screen-components/login/login.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  x = document.getElementById("login");
+  y = document.getElementById("register");
+  z = document.getElementById("btn");
```

Comment:
Angular doesn't like it when we do DOM manipulation in a component. If you ever have DOM manipulation to do (which isn't common) you can use Angular Directives to do that. In this case, it's just adding style, this should instead just be CSS. For example you're adding a style to the login div so you would instead add the css.

```css
#login {
    left: -400px;
}
```

If you need to change the style conditionally, you should be using `*ngIf` on the element or `[class.my-class-name]="condition"` to add a class conditionally and add the CSS you want to change it to in the CSS file.

you can watch more on these in the Angular fundamentals Pluralsight course or in the Angular Tour of Heroes tutorial specifically in this section: https://angular.io/tutorial/toh-pt2#style-the-selected-hero